docs/source/markdown/podman-commit.1.md

```diff
@@ -9,58 +9,52 @@ podman\-commit - Create new image based on the changed container
-configured with the `--change` flag and a commit message can be set using the
-`--message` flag. The container and its processes are paused while the image is
-committed. This minimizes the likelihood of data corruption when creating the new
-image. If this is not desired, the `--pause` flag can be set to false. When the commit
-is complete, Podman will print out the ID of the new image.
+**podman commit** creates an image based on a changed *container*. The author of the image can be set using the **--author** OPTION. Various image instructions can be configured with the **--change** OPTION and a commit message can be set using the **--message** OPTION. The *container* and its processes are paused while the image is committed. This minimizes the likelihood of data corruption when creating the new image. If this is not desired, the **--pause** OPTION can be set to *false*. When the commit is complete, Podman will print out the ID of the new image.
 
-If *image* does not begin with a registry name component, `localhost` will be added to the name.
-If *image* is not provided, the values for the `REPOSITORY` and `TAG` values of the created image will each be set to `<none>`.
+If `image` does not begin with a registry name component, `localhost` will be added to the name.
+If `image` is not provided, the values for the `REPOSITORY` and `TAG` values of the created image will each be set to `<none>`.
 
 ## OPTIONS
 
 #### **--author**, **-a**=*author*
 
-Set the author for the committed image
+Set the author for the committed image.
 
 #### **--change**, **-c**=*instruction*
 
 Apply the following possible instructions to the created image:
 **CMD** | **ENTRYPOINT** | **ENV** | **EXPOSE** | **LABEL** | **ONBUILD** | **STOPSIGNAL** | **USER** | **VOLUME** | **WORKDIR**
 
-Can be set multiple times
+Can be set multiple times.
 
-#### **--format**, **-f**=*format*
+#### **--format**, **-f** =**oci** | *docker*
 
-Set the format of the image manifest and metadata.  The currently supported formats are _oci_ and _docker_.  If
-not specifically set, the default format used is _oci_.
+Set the format of the image manifest and metadata.  The currently supported formats are **oci** and *docker*. The default is **oci**.
 
 #### **--iidfile**=*ImageIDfile*
 
 Write the image ID to the file.
 
-#### **--include-volumes**
+#### **--include-volumes** [=**false** | *true*]
 
-Include in the committed image any volumes added to the container by the `--volume` or `--mount` options to the `podman create` and `podman run` commands.
+Include in the committed image any volumes added to the container by the **--volume** or **--mount** OPTIONS to the [**podman create**](podman-create.1.md) and [**podman run**](podman-run.1.md) commands.
 
 #### **--message**, **-m**=*message*
 
-Set commit message for committed image.  The message field is not supported in _oci_ format.
+Set commit message for committed image.\
```

Comment:
This line break "\" ends up showing up in the man pages.

Reply:
The only reason i used `\` instead of the normal way, which is putting 2 spaces at the end of the line and adding a new line, is because the Validate fedora-34 Build test gives me an error because of whitespaces at the end of a line. `go-md2man` only accepts line breaks in the form of whitespace in the end and a new line. So it should be impossible to have a line break at this point. If there is truely no option for it, I will change it in general and come up with something new.